airflow/providers/google/CHANGELOG.rst

```diff
@@ -23,6 +23,47 @@
 Changelog
 ---------
 
+8.3.0
+.....
+
+Features
+~~~~~~~~
+
+* ``add description method in BigQueryCursor class (#25366)``
+* ``Add project_id as a templated variable in two BQ operators (#24768)``
+* ``Remove deprecated modules in Amazon provider (#25543)``
```

Comment:
amazon commit in gcp provider ?

Reply:
Yep. some (typing) changes in Google Provider were done as part of the Amazon change (see yoursefl).